apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -30,7 +30,7 @@ export default class ResultDetail extends React.Component<Props> {
           input: {
             bookingToken,
             bags: 0, //  @TODO - use data from passengers forms
-            passengers: { adults, infants },
+            passengers: { adults: 1, infants: 0 },
```

Comment:
This is basically used for the first check after the user clicks on the book button on the result list page, it should not be hardcoded. This data should data what user added to search from.

Reply:
True that! will put it back now!

Reply:
Done!

Reply:
It still not working as is expected, when I set adults on 6 and try to book the flight, then check the Graphql API call log, I realized that it calls with the wrong count of passenger it is always with one. Also now in url of result detail page is missing information about the count of passengers, like was before. 

<img width="718" alt="Screen Shot 2019-05-22 at 16 30 49" src="https://user-images.githubusercontent.com/6354326/58183162-20a02d00-7caf-11e9-81e4-72573bbb14d8.png">

Reply:
@FilipMessa Forgot to add it back to the router navigation, now it should work.

Reply:
I am on it 🚶